src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,435 @@
+  <figcaption class="w-figcaption">Chromium blog post with red boxes around elements with an <code>id</code> attribute.</figcaption>
+</figure>
+
+You are probably asking yourself what all the red boxes mean.
+They are the result of running the following snippet in DevTools.
+It highlights all elements that have an `id` attribute.
+
+```js
+document.querySelectorAll('[id]').forEach((el) => {
+  el.style.border = 'solid 2px red';
+});
+```
+
+I can place a deep link to any element highlighted with a red box thanks to the
+[fragment identifier](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web#Fragment)
+which I then use in the [hash](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash)
+of the page's URL.
+Assuming I wanted to deep link to the *Give us feedback in our
+[Product Forums](http://support.google.com/bin/static.py?hl=en&page=portal_groups.cs)*
+box in the aside, I could do so by handcrafting the URL
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<mark class="highlight-line highlight-line-active">#HTML1</mark></code></a>.
+As you can see in the Elements panel of the Developer Tools, the element in question
+has an `id` attribute with the value `HTML1`.
+
+<figure class="w-figure">
+  <img src="id-html1.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Dev Tools showing the <code>id</code> of an element.</figcaption>
+</figure>
+
+If I parse this URL with JavaScript's `URL()` constructor, the different components are revealed.
+Notice the `hash` property with the value `#HTML1`.
+
+```js/3
+new URL('https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1');
+/* Creates a new `URL` object
+URL {
+  hash: "#HTML1"
+  host: "blog.chromium.org"
+  hostname: "blog.chromium.org"
+  href: "https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1"
+  origin: "https://blog.chromium.org"
+  password: ""
+  pathname: "/2019/12/chrome-80-content-indexing-es-modules.html"
+  port: ""
+  protocol: "https:"
+  search: ""
+  searchParams: URLSearchParams {}
+  username: ""
+}
+*/
+```
+
+The fact though that I had to open the Developer Tools to find the `id`
+of an element speaks volumes about the probability this particular section of the page
+was meant to be linked to by the author of the blog post.
+
+What if I want to link to something without an `id`?
+Say I want to link to the *ECMAScript Modules in Web Workers* heading.
+As you can see in the screenshot below, the `<h1>` in question does not have an `id` attribute,
+meaning there is no way I can link to this heading.
+This is the problem that Text Fragments solve 🎉.
+
+<figure class="w-figure">
+  <img src="id-missing.png" alt="" class="w-screenshot" width="400">
```

Comment:
This image is also small on account of `width="400"`

![image](https://user-images.githubusercontent.com/4713486/81129002-b8fee400-8ef7-11ea-9a9e-a8e58037c24c.png)